numba/dispatcher.py

```diff
@@ -353,9 +377,13 @@ def typeof_pyval(self, val):
         """
         # Not going through the resolve_argument_type() indirection
         # can shape a couple µs.
-        tp = typeof(val)
-        if tp is None:
+        try:
```

Comment:
Above comment https://github.com/numba/numba/pull/1948/files#diff-66adccf97cb99c68d3c33d8025531cc6R379 should probably say `save` not `shape`?!